grails-app/services/com/unifina/service/DataUnionJoinRequestService.groovy

```diff
@@ -38,9 +39,14 @@ class DataUnionJoinRequestService {
 
 	protected Set<Stream> findStreams(DataUnionJoinRequest c) {
 		log.debug(String.format("entering findStreams(%s)", c))
-		List<Product> products = Product.withCriteria {
```

Comment:
Why are you refactoring `withCriteria` to `createCriteria().list`? Isn't that more or less the same thing?

Reply:
At the first look of the code I had trouble understanding what's the difference between:

`Product.createCriteria`

`Product.createCriteria()`

`Product.createCriteria().list`

Answer is none. So it's better to write out all the code to make it easy to read.

withCriteria is overloaded (doing two things) so it's difficult to read.

Reply:
Ok